simulator/infra/wide_types.h

```diff
@@ -25,7 +25,7 @@ using uint128 = unsigned __int128;
 #else // __SIZEOF_INT128__
 
 #include <boost/multiprecision/cpp_int.hpp>
-
+#include <boost/integer.hpp>
```

Comment:
Could you please explain?

Reply:
We would like to use this library for change a number of bits but so far it is not necessary.

Reply:
Ok. I advise you to set my `sign_t` and `unsign_t` metafunctions.